### ISSUE TITLE
[URGENT] Failed to deploy to Google Play: The Android App Bundle was signed with the wrong key

### ISSUE THREAD

**Issue by giladresisi** [FlutterFlow/flutterflow-issues#3314]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I'm failing to deploy to Google Play, even when I'm using a keystore with the same SHA1 signatures as in my Google Play project. I've let Google create the key for me, downloaded the .der file containing from Google Play console, converted it into a .jsk file and uploaded it to FlutterFlow when using the "Upload Keystore" advanced option in Google Play options. However it seems every time I try (and fail) to deploy to Google Play I get the above error with a different "Found: SHA1" signature, as if I didn't toggle the "Upload Keystore" option (I also verified that my .jsk file contains the same SHA1 as in the "expected: SHA1" message.

*****
Update:
I really need this fixed ASAP, my users are expecting a version which I cannot create now, this is a blocker for my work.

### Expected Behavior

I'd expect FlutterFlow to identify the SHA1 on my uploaded key instead of generating a new one, when I toggle the "Upload Keystore" option

### Steps to Reproduce

1. Deploy to Google Play

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4oz8r13ItOxLhc+KrAdfpVnSQ7Qkc3argo0MpBZywgfZzzBLN/fPfQPhdtOfXkTFFqJE6Mp14K3PTOivL2Gu5fKU2Zb7lxzpZMTBbdQlmXR4uTF7qOeHJQO99gI0CFy6ungSJAH/VtR00tw12mOeyRHwrbQJTUPWo8Sq/LZO4=

### Visual documentation

[‏‏Text Document חדש.txt](https://github.com/user-attachments/files/16095295/Text.Document.txt)

### Environment

```markdown
- FlutterFlow version: v4.1.65+
- Platform: Web
- Browser name and version: Chrome 126.0.6478.127
- Operating system and version affected: Windows 11 Pro
```

### Additional Information

Reach out in private so I can send you the keystore and .der file from Google Play console

**Comment by Alezanello** [comment]
Hello!

First of all, thanks for bringing this to our attention!

To resolve the issue you're encountering with deploying to Google Play despite using the correct keystore, follow these steps to ensure your keystore and signing configurations are correctly set up:

1. **Verify SHA1 Signatures**:
   * Ensure that the SHA1 signature of the keystore you generated matches the SHA1 signature in the Google Play Console.
   * You can verify the SHA1 signature using the following command:


   * `keytool -list -v -keystore your-keystore.jks -alias your-alias`
   * Compare the output with the SHA1 in the Google Play Console.
2. **Check the Key Properties**:
   * Make sure the alias and password for your keystore match what you’ve set in your FlutterFlow project.
3. **Ensure the Correct Keystore is Uploaded**:
   * Double-check that you uploaded the correct `.jks` file to FlutterFlow.
   * Sometimes, there might be a mix-up with files, especially if you have multiple keystore files.
4. **Google Play Console Configuration**:
   * Go to the Google Play Console.
   * Navigate to the App Signing section.
   * Ensure that the App Signing key certificate matches the certificate used in your keystore.
5. **Convert .der to .jks Properly**:
   * Ensure the conversion from `.der` to `.jks` was done correctly. The steps should look like this:


   * `keytool -importcert -file your-certificate.der -keystore your-keystore.jks -alias your-alias`
6. **Debugging the Deployment**:
   * If the deployment fails, check the logs to understand what SHA1 is being found and expected. This can provide clues about where the mismatch is occurring.
   * Ensure the package name in your app matches what is registered in the Google Play Console.

If you have followed these steps and still face issues, please provide more details on the error message you are receiving, especially the "Found: SHA1" and "Expected: SHA1" values. This information can help further diagnose the problem.

In case this doesn't help, I strongly recommend you to chat with support via the chat in the app or at support@flutterflow.io as they can help you in this specific case.

Best regards,<br>Azanelo

**Comment by giladresisi** [comment]
Got it, thanks Azanelo.

Hi support,
Please see the above urgent issue I'm facing.
I followed the exact steps suggested by Azanelo and I still encounter this
issue. The SHA1in the .jsk file I created from the.der I downloaded from
Google Play Console is what I'd expect, yet what FlutterFlow does when I
upload this .jsk is unexpected.
Every time the "Found: SHA1" is different, suggesting that FlutterFlow
creates a new key every run (as if I didn't toggle the "Upload Keystore"
option), and the "Expected: SHA1" is the same as in my app in Google Play
Console, suggesting that I downloaded the .der correctly and my .jsk
contains the correct SHA1 indeed.
To summarize - it looks like FlutterFlow doesn't notice that I toggled the
"Upload Keystore" option, please look in that direction.
Fixing this is really urgent for me as my users have been awaiting a new
version for several days now.
Thanks,
Gilad

‫בתאריך יום ה׳, 4 ביולי 2024 ב-20:29 מאת ‪Alezanello‬‏ <‪
***@***.***‬‏>:‬

**Comment by giladresisi** [comment]
Hi,
1. https://app.flutterflow.io/project/the-one-lead-7s1re6
2. Mentioned in the correspondence (tl;dr - seems as if FlutterFlow doesn't
notice I toggled the "Upload Keystore" option ON, it ignores my uploaded
key that contains the correct SHA1 and instead generates a new key with a
different SHA1 every deployment attempt).
3. I permit testing
4. See attached error message I got.
Thanks,
Gilad

‫בתאריך יום ה׳, 4 ביולי 2024 ב-21:59 מאת ‪FlutterFlow from FlutterFlow‬‏ <‪
***@***.***‬‏>:‬

> Thank you for contacting us! One of our support team members will get back
> to you as soon as possible.
>
> *If your inquiry is related to a project issue, please reply to this email
> with: *
>
>    1. Project URL (the page you were working on when you first faced this
>    issue)
>    2. Detailed description
>    3. Permission to test (yes or no)
>    4. Screenshot of the issue (if applicable)
>
> Providing this information will allow the team to dive into the issue
> right away. We'll talk to you soon!
>
> Cheers,
>
> FlutterFlow Support
>
> *FlutterFlow*
>
> FlutterFlow Support
>
> *Our team operates Monday - Friday*
> [image: intercom]
>

<body style="background-color:#14161C">
<style>
body {
  color:#C8C8C8;
  overflow-x: hidden;
  padding-top: 0px;
  padding-right: 5px;
  padding-bottom: 5px;
  padding-left: 5px;
  font-family: monaco;
}
h4 {
  margin-bottom: 4px;
  font-weight: bold;
  color: #DF3F3F;
}
</style>
<h4>Failed Step: Publishing</h4>
<div style="white-space: pre-wrap;line-height: 1.5">
== Gathering artifacts ==

== Publishing artifacts ==

Publishing artifact app-release.apk
Publishing artifact app-release.apk
Publishing artifact app-release.aab
Publishing artifact release_mapping_mapping.txt

Publishing app-release.aab to Google Play
Google Play failed to upload artefacts. The Android App Bundle was signed with the wrong key. Found: SHA1: 54:71:66:C2:B5:05:09:1B:0A:57:3A:13:F7:EA:C8:2C:40:BB:FE:8A, expected: SHA1: 2D:1A:3C:6F:6F:67:CF:B5:C9:4D:27:8A:8F:1A:84:27:15:08:24:8C.: {
    "error": {
        "code": 403,
        "message": "The Android App Bundle was signed with the wrong key. Found: SHA1: 54:71:66:C2:B5:05:09:1B:0A:57:3A:13:F7:EA:C8:2C:40:BB:FE:8A, expected: SHA1: 2D:1A:3C:6F:6F:67:CF:B5:C9:4D:27:8A:8F:1A:84:27:15:08:24:8C.",
        "status": "PERMISSION_DENIED"
    }
}

Build failed :|


Publishing failed :|
Google Play failed to upload artefacts. The Android App Bundle was signed with the wrong key. Found: SHA1: 54:71:66:C2:B5:05:09:1B:0A:57:3A:13:F7:EA:C8:2C:40:BB:FE:8A, expected: SHA1: 2D:1A:3C:6F:6F:67:CF:B5:C9:4D:27:8A:8F:1A:84:27:15:08:24:8C.: {
    "error": {
        "code": 403,
        "message": "The Android App Bundle was signed with the wrong key. Found: SHA1: 54:71:66:C2:B5:05:09:1B:0A:57:3A:13:F7:EA:C8:2C:40:BB:FE:8A, expected: SHA1: 2D:1A:3C:6F:6F:67:CF:B5:C9:4D:27:8A:8F:1A:84:27:15:08:24:8C.",
        "status": "PERMISSION_DENIED"
    }
}
</div>
</body>

**Comment by giladresisi** [comment]
Hi Thomas,
Yes, tried right now and still experiencing the same issue, can't deploy to
Google Play.
Please help.
Thanks,
Gilad

‫בתאריך שבת, 6 ביולי 2024 ב-1:37 מאת ‪Thomas from FlutterFlow‬‏ <‪
***@***.***‬‏>:‬

> Hey Gilad, 👋
> ​﻿
> ﻿How are you doing today?
> ​﻿
> ﻿I am Thomas from the support team and I will be happy to help you with
> your questions today!
> ​﻿
> ﻿First of all, thank you for choosing FlutterFlow to build your product!
> So excited to see what you build :)
> ​﻿
> I am extremely sorry for the delayed response! 😕
>
>
> ﻿This is not the experience we want you to have
>
>
> There has been a huge influx of support tickets and it's taking longer to
> get back to you than usual!
> ﻿​﻿
> ﻿Our team is working on making this experience better for you.
> ﻿​﻿
> ﻿I'm here to help you with this issue – *can you please confirm if you
> are still facing this issue?*
> ﻿​﻿
> ﻿Best,
> ﻿Thomas
>
> *Thomas*
>
> FlutterFlow Support
>
> *Our team operates Monday - Friday*
> On Thu, Jul 04, 2024 at 21:29 PM, Gilad Resisi ***@***.***>
> wrote:
>
> Hi,
> 1. https://app.flutterflow.io/project/the-one-lead-7s1re6
> 2. Mentioned in the correspondence (tl;dr - seems as if FlutterFlow
> doesn't notice I toggled the "Upload Keystore" option ON, it ignores my
> uploaded key that contains the correct SHA1 and instead generates a new key
> with a different SHA1 every deployment attempt).
> 3. I permit testing
> 4. See attached error message I got.
> Thanks,
> Gilad
>
> ‫בתאריך יום ה׳, 4 ביולי 2024 ב-21:59 מאת ‪FlutterFlow from FlutterFlow‬‏ <‪
> ***@***.***‬‏>:‬
>
> Thank you for contacting us! One of our support team members will get back
> to you as soon as possible.
>
> *If your inquiry is related to a project issue, please reply to this email
> with: *
>
>    1. Project URL (the page you were working on when you first faced this
>    issue)
>    2. Detailed description
>    3. Permission to test (yes or no)
>    4. Screenshot of the issue (if applicable)
>
> Providing this information will allow the team to dive into the issue
> right away. We'll talk to you soon!
>
> Cheers,
>
> FlutterFlow Support
>
> *FlutterFlow*
>
> FlutterFlow Support
>
> *Our team operates Monday - Friday*
> [image: intercom]
>
> [image: intercom]
>

**Comment by giladresisi** [comment]
Hi,
That's an existing app, it's not the first time I deploy it.
And why was the issue closed?? It wasn't yet resolved!
Please confirm it's still undergoing investigation.
Thanks,
Gilad

בתאריך יום ג׳, 9 ביולי 2024, 4:16, מאת Alezanello ***@***.***

**Comment by giladresisi** [comment]
Why was the issue closed??? I still have the problem!

**Comment by giladresisi** [comment]
Why was the issue closed??? I still have the problem!

‫בתאריך יום ג׳, 9 ביולי 2024 ב-20:45 מאת ‪FlutterFlow from FlutterFlow‬‏ <‪
***@***.***‬‏>:‬

> Hi Gilad! I hope you're having a great day. 😊
>
> We haven't heard back from you, so we wanted to check in and see how
> things are going. If you have any updates or questions, we're here and
> happy to help.
>
> Or if your issue is all sorted out, just drop us a quick message.
> ​
> Best,
>
> FlutterFlow Team
>
> *FlutterFlow*
>
> FlutterFlow Support
>
> *Our team operates Monday - Friday*
> [image: intercom]
>

**Comment by Alezanello** [comment]
Hello!

We didn't close the issue because it was solved; we closed it because it is more of a support request than a replicable bug from a blank project. This issue tracker is intended for identifying and resolving reproducible bugs.

Thank you for your understanding.

Best regards,<br>Azanello

**Comment by giladresisi** [comment]
So How do we solve this problem? As of now I can't deploy to Google Play!
How do I know you guys are still on this and try to resolve ASAP?

‫בתאריך יום ג׳, 9 ביולי 2024 ב-21:12 מאת ‪Alezanello‬‏ <‪
***@***.***‬‏>:‬

**Comment by Alezanello** [comment]
With your direct contact with support via in-app chat or email, we take every case very seriously and strive to fix issues promptly!

Please check your email for any support messages and continue troubleshooting this issue with them.

**Comment by giladresisi** [comment]
Hi Thomas,
I'm deploying from a different Github branch, could that be the cause for
the issue?
Please see the attached screenshot from the flutterflow branch of my
project in github.
Looking at this now I see that the "release" buildType references the
"debug" signingConfig but since there's no such signingConfig in the
build.gradle file then probably FlutterFlow generates its own key on every
attempt to build the "release" buildType, regardless of whether I uploaded
my own key or not (I assume that's the key the "release" signingConfig
references, right?).
Can you guys please advise on this ASAP so I can finally continue deploying
to Google Play?
Thanks,
Gilad


‫בתאריך יום ד׳, 10 ביולי 2024 ב-15:26 מאת ‪Thomas from FlutterFlow‬‏ <‪
***@***.***‬‏>:‬

> Hey, this issue is still open! Please be rest assured - we are here to
> help!
>
> Are you deploying with source as FlutterFlow or from Github?
>
> *Thomas*
>
> FlutterFlow Support
>
> *Our team operates Monday - Friday*
> On Tue, Jul 09, 2024 at 19:32 PM, Gilad Resisi ***@***.***>
> wrote:
>
> Why was the issue closed??? I still have the problem!
>
> ‫בתאריך יום ג׳, 9 ביולי 2024 ב-20:45 מאת ‪FlutterFlow from FlutterFlow‬‏ <‪
> ***@***.***‬‏>:‬
>
> Hi Gilad! I hope you're having a great day. 😊
>
> We haven't heard back from you, so we wanted to check in and see how
> things are going. If you have any updates or questions, we're here and
> happy to help.
>
> Or if your issue is all sorted out, just drop us a quick message.
> ​
> Best,
>
> FlutterFlow Team
>
> *FlutterFlow*
>
> FlutterFlow Support
>
> *Our team operates Monday - Friday*
> [image: intercom]
>
> [image: intercom]
>

**Comment by giladresisi** [comment]
I tried that and got a different error now, see the attached.
I see the error is:
> Failed to read key ******** from store "/tmp/keystore.keystore": trusted
certificate entries are not password-protected
Could this mean the keystore file wasn't found?
I also see that the release signingConfigs properties come from the
keystoreProperties object which is initialized by reading a
"key.properties" file supposedly located in the rootProject folder.
The question is - should I add my Upload Keystore file to the github branch
I'm using for deployment and create a file named "key.properties"
containing the file's properties? Your FAQ link doesn't mention anything
about this.
Thanks,
Gilad

‫בתאריך יום ד׳, 10 ביולי 2024 ב-16:04 מאת ‪Thomas from FlutterFlow‬‏ <‪
***@***.***‬‏>:‬

> Yes. Please change it to release and then try again. If it’s in debug, it
> will get signed with a debug key which will cause this issue.
>
> Please see our docs for more info
>
>
> https://docs.flutterflow.io/settings-and-integrations/app-settings/mobile-deployment/google-play-store-deployment#faqs
>
> I hope this helps
>
> *Thomas*
>
> FlutterFlow Support
>
> *Our team operates Monday - Friday*
> [image: intercom]
>

**Comment by giladresisi** [comment]
Found the answer:
The .jks file I tried to use was created from the .der certificate I
downloaded from Google Play Console (I lost my upload key so I found how to
create a .jks file from that .der file) but that generated .jks wasn't a
real keystore file with a private key since Google Play Console only allows
you to download the public key so the .jks was insufficient and thus the
signing failed.
I had to re-create a new .jks using keytool and then the signing and the
building of an .aab succeeded locally, I then uploaded the new .jks to
FlutterFlow, exported its content to a .pem file using keytool and uploaded
it to Google Play Console to update the upload key, the problem is that it
takes extra 2 days for them to finish the change so now I'm waiting...
I'm pretty confident the build and deployment will succeed now via
FlutterFlow, I'll update here.
Anyway, I suggest you guys update that FAQ per what I just described, and
also I guess you better inform users that they should download and keep the
initial upload keystore you guys generate for them upon the 1st deployment,
so that they can use it in case the key inside FlutterFlow is somehow lost,
and explain to them how to manually upload the keystore they downloaded.
Thanks,
Gilad

‫בתאריך יום ד׳, 10 ביולי 2024 ב-16:31 מאת ‪Gilad Resisi‬‏ <‪
***@***.***‬‏>:‬

> I tried that and got a different error now, see the attached.
> I see the error is:
> > Failed to read key ******** from store "/tmp/keystore.keystore": trusted
> certificate entries are not password-protected
> Could this mean the keystore file wasn't found?
> I also see that the release signingConfigs properties come from the
> keystoreProperties object which is initialized by reading a
> "key.properties" file supposedly located in the rootProject folder.
> The question is - should I add my Upload Keystore file to the github
> branch I'm using for deployment and create a file named "key.properties"
> containing the file's properties? Your FAQ link doesn't mention anything
> about this.
> Thanks,
> Gilad
>
> ‫בתאריך יום ד׳, 10 ביולי 2024 ב-16:04 מאת ‪Thomas from FlutterFlow‬‏ <‪
> ***@***.***‬‏>:‬
>
>> Yes. Please change it to release and then try again. If it’s in debug, it
>> will get signed with a debug key which will cause this issue.
>>
>> Please see our docs for more info
>>
>>
>> https://docs.flutterflow.io/settings-and-integrations/app-settings/mobile-deployment/google-play-store-deployment#faqs
>>
>> I hope this helps
>>
>> *Thomas*
>>
>> FlutterFlow Support
>>
>> *Our team operates Monday - Friday*
>> [image: intercom]
>>
>

**Comment by giladresisi** [comment]
Please just don't close the ticket yet, wait until I deploy to Google Play
successfully from FlutterFlow (the change of upload keystore in Google is
scheduled to finish tomorrow around 11:30 am UTC, shortly afterwards I'll
probably try to deploy and update here).
Thanks,
Gilad

‫בתאריך יום ה׳, 11 ביולי 2024 ב-15:48 מאת ‪Thomas from FlutterFlow‬‏ <‪
***@***.***‬‏>:‬

> Hey, thanks for the update!
>
> I will pass this feedback about the docs update to the team!
>
>
> Is there anything else I can help you with today?
> ﻿​﻿
> ﻿Best,
> ﻿Thomas
>
> *Thomas*
>
> FlutterFlow Support
>
> *Our team operates Monday - Friday*
> On Wed, Jul 10, 2024 at 17:25 PM, Gilad Resisi ***@***.***>
> wrote:
>
> Found the answer:
> The .jks file I tried to use was created from the .der certificate I
> downloaded from Google Play Console (I lost my upload key so I found how to
> create a .jks file from that .der file) but that generated .jks wasn't a
> real keystore file with a private key since Google Play Console only allows
> you to download the public key so the .jks was insufficient and thus the
> signing failed.
> I had to re-create a new .jks using keytool and then the signing and the
> building of an .aab succeeded locally, I then uploaded the new .jks to
> FlutterFlow, exported its content to a .pem file using keytool and uploaded
> it to Google Play Console to update the upload key, the problem is that it
> takes extra 2 days for them to finish the change so now I'm waiting...
> I'm pretty confident the build and deployment will succeed now via
> FlutterFlow, I'll update here.
> Anyway, I suggest you guys update that FAQ per what I just described, and
> also I guess you better inform users that they should download and keep the
> initial upload keystore you guys generate for them upon the 1st deployment,
> so that they can use it in case the key inside FlutterFlow is somehow lost,
> and explain to them how to manually upload the keystore they downloaded.
> Thanks,
> Gilad
>
> ‫בתאריך יום ד׳, 10 ביולי 2024 ב-16:31 מאת ‪Gilad Resisi‬‏ <‪
> ***@***.***‬‏>:‬
>
> I tried that and got a different error now, see the attached.
> I see the error is:
> > Failed to read key ******** from store "/tmp/keystore.keystore": trusted
> certificate entries are not password-protected
> Could this mean the keystore file wasn't found?
> I also see that the release signingConfigs properties come from the
> keystoreProperties object which is initialized by reading a
> "key.properties" file supposedly located in the rootProject folder.
> The question is - should I add my Upload Keystore file to the github
> branch I'm using for deployment and create a file named "key.properties"
> containing the file's properties? Your FAQ link doesn't mention anything
> about this.
> Thanks,
> Gilad
>
> ‫בתאריך יום ד׳, 10 ביולי 2024 ב-16:04 מאת ‪Thomas from FlutterFlow‬‏ <‪
> ***@***.***‬‏>:‬
>
> Yes. Please change it to release and then try again. If it’s in debug, it
> will get signed with a debug key which will cause this issue.
>
> Please see our docs for more info
>
>
> https://docs.flutterflow.io/settings-and-integrations/app-settings/mobile-deployment/google-play-store-deployment#faqs
>
> I hope this helps
>
> *Thomas*
>
> FlutterFlow Support
>
> *Our team operates Monday - Friday*
>
> [image: intercom]
>

**Comment by giladresisi** [comment]
Hi, this time it worked!
I'm sure many developers stumble into this issue like me, please update the
documentation and the explanations within FlutterFlow to help avoid these
cases. I can explain again in detail if you want me to.
Thanks for your help!
Gilad

בתאריך יום ה׳, 11 ביולי 2024, 20:12, מאת Thomas from FlutterFlow ‏<
***@***.***>:

> Sure, no issues :)
>
> I will keep this open and wait for your update!
>
> Best
> Thomas
>
> *Thomas*
>
> FlutterFlow Support
>
> *Our team operates Monday - Friday*
> On Thu, Jul 11, 2024 at 15:37 PM, Gilad Resisi ***@***.***>
> wrote:
>
> Please just don't close the ticket yet, wait until I deploy to Google Play
> successfully from FlutterFlow (the change of upload keystore in Google is
> scheduled to finish tomorrow around 11:30 am UTC, shortly afterwards I'll
> probably try to deploy and update here).
> Thanks,
> Gilad
>
> ‫בתאריך יום ה׳, 11 ביולי 2024 ב-15:48 מאת ‪Thomas from FlutterFlow‬‏ <‪
> ***@***.***‬‏>:‬
>
> Hey, thanks for the update!
>
> I will pass this feedback about the docs update to the team!
>
>
> Is there anything else I can help you with today?
> ﻿​﻿
> ﻿Best,
> ﻿Thomas
>
> *Thomas*
>
> FlutterFlow Support
>
> *Our team operates Monday - Friday*
>
> On Wed, Jul 10, 2024 at 17:25 PM, Gilad Resisi ***@***.***>
> wrote:
>
> Found the answer:
> The .jks file I tried to use was created from the .der certificate I
> downloaded from Google Play Console (I lost my upload key so I found how to
> create a .jks file from that .der file) but that generated .jks wasn't a
> real keystore file with a private key since Google Play Console only allows
> you to download the public key so the .jks was insufficient and thus the
> signing failed.
> I had to re-create a new .jks using keytool and then the signing and the
> building of an .aab succeeded locally, I then uploaded the new .jks to
> FlutterFlow, exported its content to a .pem file using keytool and uploaded
> it to Google Play Console to update the upload key, the problem is that it
> takes extra 2 days for them to finish the change so now I'm waiting...
> I'm pretty confident the build and deployment will succeed now via
> FlutterFlow, I'll update here.
> Anyway, I suggest you guys update that FAQ per what I just described, and
> also I guess you better inform users that they should download and keep the
> initial upload keystore you guys generate for them upon the 1st deployment,
> so that they can use it in case the key inside FlutterFlow is somehow lost,
> and explain to them how to manually upload the keystore they downloaded.
> Thanks,
> Gilad
>
> ‫בתאריך יום ד׳, 10 ביולי 2024 ב-16:31 מאת ‪Gilad Resisi‬‏ <‪
> ***@***.***‬‏>:‬
>
> I tried that and got a different error now, see the attached.
> I see the error is:
> > Failed to read key ******** from store "/tmp/keystore.keystore": trusted
> certificate entries are not password-protected
> Could this mean the keystore file wasn't found?
> I also see that the release signingConfigs properties come from the
> keystoreProperties object which is initialized by reading a
> "key.properties" file supposedly located in the rootProject folder.
> The question is - should I add my Upload Keystore file to the github
> branch I'm using for deployment and create a file named "key.properties"
> containing the file's properties? Your FAQ link doesn't mention anything
> about this.
> Thanks,
> Gilad
>
> ‫בתאריך יום ד׳, 10 ביולי 2024 ב-16:04 מאת ‪Thomas from FlutterFlow‬‏ <‪
> ***@***.***‬‏>:‬
>
> Yes. Please change it to release and then try again. If it’s in debug, it
> will get signed with a debug key which will cause this issue.
>
> Please see our docs for more info
>
>
> https://docs.flutterflow.io/settings-and-integrations/app-settings/mobile-deployment/google-play-store-deployment#faqs
>
> I hope this helps
>
> *Thomas*
>
> FlutterFlow Support
>
> *Our team operates Monday - Friday*
>
> [image: intercom]
>
> [image: intercom]
>
